### DISCUSSION TITLE
Why did you remove "Quick Bookmark" and replaced with "Share Post"?

### DISCUSSION THREAD

**Discussion by VickiiHub** [dailydotdev/daily#757] [Feature Requests]
Why did you remove quick bookmark and replaced with "Share Post" there is enough area to for both "Quick Bookmark" and "Share Post" button.
![Screen Shot 2022-10-30 at 22 07 50](https://user-images.githubusercontent.com/5210322/198890361-18bc7182-3366-40dd-9049-f744a507477e.png)

**Comment by idoshamun** [comment]
Hi,

We've tried several combinations to make the bookmark and share live together on the card. 
Unfortunately, we were not able to find a visually appealing combo for this, they all look very crowded in the feed.

**Reply by MystPi** [reply]
I personally would rather have the bookmark button.

**Reply by VickiiHub** [reply]
Bro, there is enough space to place both bookmark and share button, cause they don't require any metrics to show alongside. At least give it try and share it to this thread. Thanks.

**Reply by idoshamun** [reply]
We have an active discussion about this matter already where I replied with the relevant solution we applied. Feel free to comment there.
https://github.com/dailydotdev/daily/discussions/638#discussioncomment-4320415

**Comment by seeniforu** [comment]
Yeah we need that bookmark button for sure.

**Comment by nimrodkra** [comment]
@VickiiHub @MystPi @seeniforu we added the bookmark button back to the card - enjoy! Thanks for the feedback 🙏

![IMG_8444](https://github.com/dailydotdev/daily/assets/41470823/3d357007-d2b8-4c2a-8a67-a6c8c92fb346)

**Reply by MystPi** [reply]
This is great! Thank you so much.
